web_src/js/components/RepoActionView.vue

```diff
@@ -404,6 +412,9 @@ export function initRepositoryActionView() {
               <a class="job-artifacts-link" target="_blank" :href="run.link+'/artifacts/'+artifact.name">
                 <SvgIcon name="octicon-file" class="ui text black job-artifacts-icon"/>{{ artifact.name }}
               </a>
+              <a v-if="run.canRerun" @click="deleteArtifact(artifact.name)" class="job-artifacts-delete">
```

Comment:
Should we use the same permission?
I mean yes, makes sense to use the same permission for it as for rerunning workflows...

Reply:
add `canDeleteArtifact` field as same permission